frontend/src/models/dashboardItemsModel.tsx

```diff
@@ -3,23 +3,34 @@ import { kea } from 'kea'
 import api from 'lib/api'
 import { prompt } from 'lib/logic/prompt'
 import { toast } from 'react-toastify'
-import { DashboardItemType } from '~/types'
+import { DashboardItemMode, DashboardItemType } from '~/types'
 import { dashboardsModel } from './dashboardsModel'
 import { Link } from 'lib/components/Link'
 import { dashboardItemsModelType } from '~/models/dashboardItemsModelType'
+import { insightLogic } from 'scenes/insights/insightLogic'
 
 export const dashboardItemsModel = kea<dashboardItemsModelType<DashboardItemType>>({
     actions: () => ({
         renameDashboardItem: (item: DashboardItemType) => ({ item }),
         renameDashboardItemSuccess: (item: DashboardItemType) => ({ item }),
+        updateDashboardItem: (id: number, payload: any) => ({ id, ...payload }),
```

Comment:
The payload could be a `Partial<DashboardItemType>` probably

Reply:
I'm getting `✖ flake8 --select=E9,F63,F7,F82 failed without output (ENOENT)` . errors when I try to merge this branch with master, how do I resolve this?

Reply:
Hmmm not sure what's going on there, but I fix the prettier error and merged master. Sometimes when you try to commit merge commits, husky will fail for some reason, you can try finishing the commit with `--no-verify`.